src/view/components/rawDataElementSelector.jsx

```diff
@@ -45,9 +44,7 @@ const RawDataElementSelector = ({
         isQuiet
         onPress={openDataElementSelector}
         aria-label="Select data element"
-        UNSAFE_className={
-          adjustForLabel ? "DataElementSelector-buttonAdjustedForLabel" : ""
-        }
+        UNSAFE_style={adjustForLabel ? { marginTop: "size-300" } : {}}
```

Comment:
This won't work because `size-300` isn't a valid CSS value. You'll need to use the `marginTop` prop. It's a prop provided by React-Spectrum that does the interpretation from `size-300` to 24px.